core/silkworm/consensus/ethash/ethash.cpp

```diff
@@ -178,23 +90,18 @@ ValidationResult validate_block_header(const BlockHeader& header, const State& s
         return ValidationResult::kWrongBaseFee;
     }
 
-    // Ethash PoW verification
-    if (config.seal_engine == SealEngineType::kEthash) {
```

Comment:
We also need to support kNoProof for consensus tests

Reply:
Oh right ... will revert. I was thinking ahead wrt the post I made now in discord

Reply:
Reverted